fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -357,6 +357,12 @@ public void setErrorCode(String errorCode) {
         this.errorCode = errorCode;
     }
 
+    public void setErrorCodeOnce(String errorCode) {
```

Comment:
Why must `once`?

Reply:
record the first error code of query, otherwise it will be overwritten by subsequent errors, such as channel